### PR TITLE
Create check-services script

### PR DESCRIPTION
**Pull request by rubo77**
reworked https://github.com/ffnord/ffnord-puppet-gateway/pull/127

This script can be used on the console of a gateway whenever you want to check if all needed services are running. It:
-  analyzes all services and processes that should run on the gateway
-  shows the maintenance status
-  shows the gateway mode

### CODE REVIEW

File: files/usr/local/bin/check-services [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# this script checks maintenance mode and gateway mode
+# and checks if those services are running:
+# isc-dhcp-server radvd ntp openvpn rpcbind fastd bind9 bird6 bird alfred batadv-vis named tincd
+
+MESH_CODE=$(ls /sys/class/net/bat-* -d|sed 's/.*\/bat-//g')
```
Comment (sargon) [inline files/usr/local/bin/check-services]: This may contain more than one result.

File: files/usr/local/bin/check-services [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+if test -e /etc/$MESH_CODE ; then 
+  . /etc/$MESH_CODE
```
Comment (sargon) [inline files/usr/local/bin/check-services]: The file containing that information is always /etc/ffnord.

File: files/usr/local/bin/check-services (hunk 1 above)
Comment (rubo77) [inline files/usr/local/bin/check-services]: how? there is only one bat-ffCOMUNITYSHORT

**Comment by rubo77** [comment]
can this be merged now?

We are using it all the time already added by hand with:

    cd /root/
    wget https://raw.githubusercontent.com/rubo77/ffnord-puppet-gateway/check-services/files/usr/local/bin/check-services
    chmod +x check-services 
    ./check-services

**Pull request merged**